empress/support_files/js/empress.js

```diff
@@ -2271,18 +2272,47 @@ define([
             obs[uniqueVal] = new Set(uniqueValueToFeatures[uniqueVal]);
         });
 
-        // assign colors to unique values
-        var colorer = new Colorer(
-            color,
-            sortedUniqueValues,
-            undefined,
-            undefined,
-            reverse
-        );
+        var colorer;
+        try {
+            // assign colors to unique values
+            colorer = new Colorer(
+                color,
+                sortedUniqueValues,
+                continuous,
+                // Colorer will create a special gradient ID using the number
+                // we pass into this parameter. This allows empress to display
+                // multiple gradients at the same time without them overriding
+                // each other. Currently, the barplots are setup to start at
+                // 0. So we set this to -1 this value will never be -1 in
+                // the barplots. Thus, by setting this value to -1 we can
+                // display the feature metadata gradient along side the
+                // barplot gradients.
+                continuous ? -1 : undefined,
+                reverse
+            );
+        } catch (err) {
+            // If the Colorer construction failed (should only have
+            // happened if the user asked for continuous values but the
+            // selected field doesn't have at least 2 unique numeric
+            // values), then we open a toast message about this error and
+            // then raise it again to send error to console.
+            var msg =
+                "Error with assigning colors: '" +
+                cat +
+                "' is either non-numeric" +
+                " or has less than 2 unique numeric values";
```

Comment:
```suggestion
                " has less than 2 unique numeric values.";
```

I don't think there is a big distinction between these cases -- all non-numeric columns have less than 2 unique numeric values, by definition. This matches the error used for continuous barplots.